docs/zilevm/developer-rpc.md

```diff
@@ -0,0 +1,74 @@
+---
+id: developer-rpc
+title: RPC
+keywords:
+  - RPC
+description: Specification of ZILEVM RPC Endpoint
+---
+
+---
+
+## RPC API specification
+
+### Fully supported APIs
+
+eth_getStorageAt
+eth_getCode
+eth_getBalance
+eth_blockNumber
+web3_clientVersion
+web3_sha3
+net_version
+net_listening
+net_peerCount
+eth_protocolVersion
+eth_chainId
+eth_mining (alway returns false)
+eth_getBlockTransactionCountByHash
+eth_getBlockTransactionCountByNumber
+eth_getUncleCountByBlockHash
+eth_getUncleCountByBlockNumber
+eth_call
+eth_getTransactionCount
+eth_getTransactionByHash
+eth_getTransactionByBlockHashAndIndex
+eth_getTransactionByBlockNumberAndIndex
+eth_sendRawTransaction
+eth_newFilter
+eth_newBlockFilter
+eth_newPendingTransactionFilter
+eth_uninstallFilter
+eth_getFilterChanges
+eth_getFilterLogs
+eth_getLogs
+eth_subscribe
+eth_unsubscribe
```

Comment:
```suggestion
* `eth_getStorageAt`
* `eth_getCode`
* `eth_getBalance`
* `eth_blockNumber`
* `web3_clientVersion`
* `web3_sha3`
* `net_version`
* `net_listening`
* `net_peerCount`
* `eth_protocolVersion`
* `eth_chainId`
* `eth_mining (alway returns false)`
* `eth_getBlockTransactionCountByHash`
* `eth_getBlockTransactionCountByNumber`
* `eth_getUncleCountByBlockHash`
* `eth_getUncleCountByBlockNumber`
* `eth_call`
* `eth_getTransactionCount`
* `eth_getTransactionByHash`
* `eth_getTransactionByBlockHashAndIndex`
* `eth_getTransactionByBlockNumberAndIndex`
* `eth_sendRawTransaction`
* `eth_newFilter`
* `eth_newBlockFilter`
* `eth_newPendingTransactionFilter`
* `eth_uninstallFilter`
* `eth_getFilterChanges`
* `eth_getFilterLogs`
* `eth_getLogs`
* `eth_subscribe`
* `eth_unsubscribe`
```

Reply:
Similar formatting should apply to the other lists. Right now the rendered version looks messy (at least in GitHub).